astropy/visualization/units.py

```diff
@@ -8,7 +8,8 @@
 __doctest_skip__ = ["quantity_support"]
 
 
-def quantity_support(format="latex_inline"):
+def quantity_support(*args,format="latex_inline"):
```

Comment:
Note that we cannot suddenly make `format` a keyword-only argument! We can at best add a `**kwargs` (or have to go through checking `args[0]` is `str` and then emit a deprecation warning).